modules/profiles/laptop-x86.nix

```diff
@@ -121,6 +121,68 @@ in {
       logging.client.endpoint = "http://${listenerAddress}:${listenerPort}/loki/api/v1/push";
       logging.listener.address = "admin-vm-debug";
       logging.listener.port = 9999;
+
+      # USB hot plugging configuration
+      hardware.usb.vhotplug = {
+        enable = true;
+        rules = [
```

Comment:
Perhaps we can move the rule configuration as this file should be a high level laptop config. Perhaps we can put this into the hardware/usb/vhotplug as default configuration?